### PR TITLE
Pr11 tests part1

### PR DESCRIPTION
**Pull request by IvanFadieiev**
Untitled

### CODE REVIEW

**Review by Zel1oy — APPROVED** [review]
GJ

File: src/test/java/project/bookstore/controller/BookControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+        String jsonRequest = objectMapper.writeValueAsString(BOOK_REQUEST_DTO);
+        MvcResult result = mockMvc.perform(post("/books")
+                        .content(jsonRequest)
+                        .contentType(MediaType.APPLICATION_JSON)
+                )
+                .andExpect(status().isCreated())
+                .andReturn();
+        BookDto actual = objectMapper.readValue(result.getResponse()
+                .getContentAsString(), BookDto.class);
+        Assertions.assertNotNull(actual);
```
Comment (andrii-hoienko) [inline src/test/java/project/bookstore/controller/BookControllerTest.java:88]: Please use static import

File: src/test/java/project/bookstore/service/BookServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+    private BookServiceImpl bookService;
+    @Mock
+    private BookRepository bookRepository;
+    @Mock
+    private BookMapper bookMapper;
+
+    @Test
+    @DisplayName("Verify create() method works")
+    public void create_ValidBookRequestDto_ReturnsBookDto() {
+        Mockito.when(bookMapper.toModel(BOOK_REQUEST_DTO)).thenReturn(BOOK);
```
Comment (andrii-hoienko) [inline src/test/java/project/bookstore/service/BookServiceTest.java:70]: static import

File: src/test/java/project/bookstore/service/BookServiceTest.java [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+
+        List<BookDto> bookDtoList = bookService.findAll(PAGEABLE);
+
+        assertEquals(bookDtoList.size(), BOOKS.size());
+    }
+
+    @Test
+    @DisplayName("Verify findAllByCategoryId() method words")
+    public void findAllByCategoryId_ValidCategoryId_ReturnsBookList() {
+        BookDtoWithoutCategoryIds bookDtoWithoutCategoryIds
```
Comment (andrii-hoienko) [inline src/test/java/project/bookstore/service/BookServiceTest.java:132]: Please all object creation code outside a method

File: src/test/resources/database/scripts/books/add-book-to-books-table.sql [hunk 4]
```diff
@@ -0,0 +1,2 @@
+INSERT INTO books (id, title, author, isbn, price, description, cover_image, is_deleted)
+VALUES (1, 'Meditations', 'Marcus Aurelius', '978-4-2596-7831-9', 1000, 'Description', 'Cover image', false);
```
Comment (andrii-hoienko) [inline src/test/resources/database/scripts/books/add-book-to-books-table.sql:2]: add empty lines

**Review by andrii-hoienko — CHANGES_REQUESTED** [review]
GJ! Let`s improve your solution

**Review by vovasalyha — APPROVED** [review]

**Pull request merged**